test-requirements.txt

```diff
@@ -0,0 +1 @@
+odoo-addon-account_financial_risk @ git+https://github.com/OCA/credit-control.git@refs/pull/272/head#subdirectory=setup/account_financial_risk
```

Comment:
Could please you rebase and update this dependency? 🙏🏻 it seems it was replaced by another PR

- https://github.com/OCA/credit-control/pull/272
- https://github.com/OCA/credit-control/pull/292